src/components/LicenseText.vue

```diff
@@ -122,7 +119,9 @@ export default {
                 workAttrs.property = 'dct:title'
             }
             if (workUrl) {
-                workAttrs.href = workUrl
```

Comment:
I think it would be cleaner to replace 'workUrl' inside the 'workProps' function with 'this.workUrl'. This way, it will be using the value from the computed. You don't need to move the code from `workUrl` to `workProps`

Reply:
Definitely, it would be cleaner and better

Reply:
Thank you for your contributions, @neeraj-2 !